internal/server/server_test.go

```diff
@@ -272,11 +277,28 @@ func TestServer_GenerateRoutes_NoRoute(t *testing.T) {
 			},
 		},
 		{
-			name: "ui path",
+			name: "text/html",
 			path: "/not/found",
 			expected: func(t *testing.T, resp *httptest.ResponseRecorder) {
 				// response should have an html body
 				assert.Assert(t, is.Contains(resp.Body.String(), "404 - example"))
+
+			},
+		},
+		{
+			name: "other (this will give plaintext)",
+			path: "/not/found",
+			expected: func(t *testing.T, resp *httptest.ResponseRecorder) {
+				// response should have an html body
```

Comment:
Aren't these giving a `text/plain` response, not html ? I guess this is just copy/paste and this comment can be removed (same in the case below).